From 392d2a8908726bc6db28b44132668e92f4f1c9a0 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Fri, 14 Jul 2017 23:13:29 +0800
Subject: [PATCH 2233/5242] MLK-16013-8 ARM64: dts: imx8mq: add
 dis_u2_susphy_quirk for usb

commit  392ca57c2dfd300acb6a7204b4c9aba2b7190553 from
https://source.codeaurora.org/external/imx/linux-imx.git

This is temp workaround to resolve usb host enumeration issue
by disable usb2 phy suspend.

Reviewed-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 7ba4f0f..f87b4cc 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -535,6 +535,7 @@
 			phy-names = "usb2-phy", "usb3-phy";
 			power-domains = <&power 2>;
 			snps,power-down-scale = <2>;
+			snps,dis_u2_susphy_quirk;
 			status = "disabled";
 		};
 	};
@@ -574,6 +575,7 @@
 			phy-names = "usb2-phy", "usb3-phy";
 			power-domains = <&power 3>;
 			snps,power-down-scale = <2>;
+			snps,dis_u2_susphy_quirk;
 			status = "disabled";
 		};
 	};
-- 
1.7.9.5

